pkg/distributor/distributor_test.go

```diff
@@ -2022,16 +2022,15 @@ func TestDistributor_MetricsMetadata(t *testing.T) {
 			_, err := ds[0].Push(ctx, req)
 			require.NoError(t, err)
 
+			// Check how many ingesters are queried as part of the shuffle sharding subring.
```

Comment:
I was going to add unit tests explicitly for `GetIngestersForMetadata` and `GetIngestersForQuery` but this test is already doing a similar verification. Swapped it to use this method instead of counting mock calls because we don't have to test both `expectedIngesters` and `expectedIngesters - 1`